jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/CalendarToolBarAssert.java

```diff
@@ -0,0 +1,40 @@
+package com.epam.jdi.light.vuetify.asserts;
+
+import com.epam.jdi.light.asserts.generic.UIAssert;
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.vuetify.elements.complex.bars.CalendarToolBar;
+import org.hamcrest.Matchers;
+import java.time.Month;
+
+import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
+
+public class CalendarToolBarAssert extends UIAssert<CalendarToolBarAssert, CalendarToolBar> {
+
+    @JDIAction("Assert that {name} has active month")
+    public CalendarToolBarAssert activeMonth(Month expectedMonth) {
+        Month actualMonth = element().activeMonth();
+        jdiAssert(
+                actualMonth,
+                Matchers.equalTo(expectedMonth),
+                String.format(
+                        "Wrong active month. Expected: %s, but was: %s",
+                        expectedMonth, actualMonth
+                )
+        );
+        return this;
+    }
+
+    @JDIAction("Assert that {name} has active year")
+    public CalendarToolBarAssert activeYear(int expectedYear) {
+        int actualYear = element().activeYear();
+        jdiAssert(
+                actualYear,
+                Matchers.equalTo(expectedYear),
+                String.format(
+                        "Wrong active year. Expected: %s, but was: %s",
+                        expectedYear, actualYear
+                )
+        );
+        return this;
+    }
+}
```

Comment:
Assert for day in one class, asserts for month and year in another. Could you, please, clarify, why such logic? :)

Reply:
According to task, we should split calendar into two separated objects. And ToolBar has month and year, when calendar itself has only day. Therefore, I need to call for a pieces of date separately.

Reply:
thanks for clarification :)